File: test-suite-netty-ssl-graalvm/src/test/java/example/micronaut/HelloController.java
package example.micronaut;

import io.micronaut.http.MediaType;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;

@Controller("/hello")
public class HelloController {

    @Get(produces = MediaType.TEXT_PLAIN)
    public String index() {
        return "Hello World";
    }
}

File: test-suite-python/src/test/python/micronaut/docs/server/intro/HelloController.py
# tag::imports[]
from micronaut.http import MediaType
from micronaut.http.annotation import Controller, Get
# end::imports[]
from micronaut.context.annotation import Requires

@Requires(missingProperty="spec.name")
# tag::class[]
@Controller("/hello") # <1>
class HelloController:
    @Get(produces=MediaType.TEXT_PLAIN) # <2>
    def index(self) -> str:
        return "Hello World" # <3>
# end::class[]
